src/targets/ruby/native.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+var util = require('util')
+
+module.exports = function (source, options) {
+  var self = this
+  var code = []
+  code.push('require \'uri\'')
```

Comment:
if you use double quotes when single quotes are used inline, the linter wont complain.